assets/schema_input.json

```diff
@@ -36,96 +36,77 @@
             "normdist": {
                 "type": "number",
                 "errorMessage": "normshift is a svbenchmark parameter. Has to be between 0-1. Disallow matches if alternate alleles have normalized edit distance greater than normdist (default 0.2)",
-                "meta": ["normdist"],
-                "default": 0.2,
-                "minimum": 0,
-                "maximum": 1
```

Comment:
You can keep these. they won't be validated if the value has not been given

Reply:
I wanted to be safe 🗡️ ahaha

Reply:
Don't worry it won't break anything :)